apps/src/templates/teacherDashboard/SectionLoginInfo.jsx

```diff
@@ -226,7 +227,6 @@ class WordOrPictureLogins extends React.Component {
     printWindow.document.write('<body onafterprint="self.close()">');
     printWindow.document.write(printArea);
     printWindow.document.write('</body></html>');
-    printWindow.print();
```

Comment:
:joy: I wonder if we'll get complaints about this.  Either way, I think it's worth a shot and we can do something more complicated around waiting for all images to load later, if we really need to.